src/Toolkit/Toolkit.WPF/UI/Controls/PopupViewer/PopupViewer.Theme.xaml

```diff
@@ -57,12 +57,33 @@
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type primitives:AttachmentsPopupElementView}">
                     <StackPanel>
+                        <StackPanel.Resources>
+                            <Style TargetType="TextBlock" x:Key="DownloadIcon">
+                                <Setter Property="FontFamily" Value="Segoe MDL2 Assets" />
+                                <Setter Property="FontSize" Value="18" />
+                                <Setter Property="VerticalAlignment" Value="Center" />
+                                <Setter Property="Margin" Value="0,0,5,0" />
+                                <Setter Property="Text" Value="&#xEBD3;" />
```

Comment:
Isn't this a [Segoe Fluent](https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font) (Windows 11+) codepoint? I don't see it in the [Segoe MDL2 Assets](https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-ui-symbol-font) character map.  I have a feeling this won't work on Win 10.

Same for the other two icons in this style.

Reply:
Segoe MDL2 Assets ships with Windows 10